tools/rna_tools/rnasnp/rnasnp.xml

```diff
@@ -37,8 +37,8 @@
         --edist=$edist
         --boltzmannPreFactor=$boltzmannPreFactor
         --winsizeFold=$mode.winsizeFold
-        > $outfile
-
+        > '$outfile'
+        ; grep "SNP" '$outfile' > /dev/null 
```

Comment:
It didn't work
"This will automatically fail the tool if the exit code is non-zero, or if the phrases error: or exception: appear in STDERR."
http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#exit-code-detection